tests/unit/test_finder.py

```diff
@@ -365,6 +366,27 @@ def test_finder_only_installs_stable_releases(data):
         assert link.url == "https://foo/bar-1.0.tar.gz"
 
 
+def test_finder_only_installs_data_require(data):
+    """
+    """
+
+    # using a local index (that has pre & dev releases)
+    finder = PackageFinder([],
+                           [data.index_url("datarequire")],
```

Comment:
I'm wondering if it would not be clearer to create the `index.html` file in the test directly ?
But I'm not sure ^^